src/helpers.js

```diff
@@ -79,21 +78,79 @@ export const getErrorIndexFromKeys = (keys) => {
     return -1;
 };
 
-export const mergeErrors = (currentErrors = {}, nextErrors = {}) => {
+const hasNoProps = (obj) => {
+    if (obj === null) {
```

Comment:
Because `true` should be returned only if given param is a non-null object which has no props. I changed this function name to `isObjectWithoutProps`, to be more clear about this condition.